paper/ms.tex

```diff
@@ -500,6 +496,7 @@ \subsubsection{Manual vetting}
       objects in Table~\ref{tab:thetable}
 			is available online {\bf For internal collaboration review:
 				\url{https://www.dropbox.com/sh/khtwk5a6z0zgrif/AABRG21Ire7VS4BUnaD1hOR8a?dl=0}}.
+			{\red (RJ: Will these be made available on Zenodo, or something with a permanent DOI?)}
```

Comment:
Yep -- it'll be a "figure set", which is one of the enhanced AAS journals figure types.